client/src/components/AddCard/AddCardForm/useStyles.ts

```diff
@@ -0,0 +1,48 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing(1),
+    alignItems: 'center',
```

Comment:
The form needs to have a flex display in order for most of these styles to work :)